util/Migrator/DbScripts/2022-11-03_00_OrganizationDomainInit.sql

```diff
@@ -0,0 +1,159 @@
+-- Create Organization Domain table
+IF OBJECT_ID('[dbo].[OrganizationDomain]') IS NOT NULL
+BEGIN
+    DROP TABLE [dbo].[OrganizationDomain]
+END
+GO
+
+IF OBJECT_ID('[dbo].[OrganizationDomain]') IS NULL
+BEGIN
+CREATE TABLE [dbo].[OrganizationDomain] (
+    [Id]                UNIQUEIDENTIFIER NOT NULL,
+    [OrganizationId]    UNIQUEIDENTIFIER NOT NULL,
+    [Txt]               VARCHAR(MAX)     NOT NULL,
+    [DomainName]        NVARCHAR(255)    NOT NULL,
+    [CreationDate]      DATETIME2(7)     NOT NULL,
+    [VerifiedDate]      DATETIME2(7)     NULL,
+    [NextRunDate]       DATETIME2(7)     NOT NULL,
+    [NextRunCount]      TINYINT          NOT NULL,
+    [Active]            BIT              NOT NULL,
+    CONSTRAINT [PK_OrganizationDomain] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_OrganzationDomain_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id])
+)
+END
+GO
+
+-- Create View
+IF EXISTS(SELECT * FROM sys.views WHERE [Name] = 'OrganizationDomainView')
```

Comment:
Can you see if we could simplify this using the `CREATE OR ALTER` syntax that's available in SQL now?  This should simplify all of the views and procedures.

Reply:
Thanks, @trmartin4 never knew about that statement